### PR TITLE
docs: update moderator handbook

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Updates the handbook to rewrite the Discord section for the new chat server, removing Discord entirely.

### CODE REVIEW

**Review by raisedadead — COMMENTED** [review]
Holding this up for a bit to coincide with the announcement.

**Comment by nhcarrigan** [comment]
Sounds good.

File: docs/moderator-handbook.md [hunk 1]
```diff
@@ -203,70 +203,62 @@ Sometimes people will post things that they think are funny. They don't realize
...
 - Someone has flooded the chat with lots of spam messages to such an extreme extent (usually involving bots) as to render chat completely unusable.
 - Someone has posted an advertisement and/or a self-promoting message/image (social media).
 
-In all other situations - even situations where the code of conduct is violated - Moderators should not delete the message as these are an important historic record. When you do delete a message, make sure you take a screenshot of it first! The screenshot can be logged in the #mod-log channel, but for the #activity-log it is sufficient to say the evidence was "removed due to sensitive content". Note: If the message contains material that would be illegal to take a screenshot of, copy the message link instead - provide that message link to @raisedadead to forward to Discord's Trust and Safety team.
+In all other situations - even situations where the code of conduct is violated - moderators should not delete the message as these are an important historic record. When you do delete a message, make sure you take a screenshot of it first! The screenshot can be logged in the #mod-log channel.
```
Comment (marsonya) [inline docs/moderator-handbook.md:234]:
```suggestion
In all other situations - even situations where the code of conduct is violated - moderators should not delete the messages as they are important historic records. When you do delete a message, make sure you take a screenshot of it first! The screenshot can be logged in the #mod-log channel.
```

File: docs/moderator-handbook.md [hunk 2]
```diff
@@ -203,70 +203,62 @@ Sometimes people will post things that they think are funny. They don't realize
...
-- The bot will create a new channel under the `private` category and add the `@username`-mentioned camper and all moderators with the `Your Friendly Moderator` role. While all moderators are added to the channel for transparency, the moderator who calls this command should be the only one to interact with the camper unless they request assistance.
-- When the conversation is complete, call the `!fCC moderate close` command _in the private channel_ to have the bot close and delete that channel.
-
-4. **Deleting messages**
-   Moderators have the ability to delete messages on Discord. They should only exercise this ability in four very specific situations:
+4. **Deleting messages** Moderators have the ability to delete messages on our chat server. They should only exercise this ability in four very specific situations:
```
Comment (marsonya) [inline docs/moderator-handbook.md:227]:
```suggestion
4. **Deleting messages** Moderators can delete messages on our chat server. They should only exercise this ability in four very specific situations:
```

**Review by marsonya — CHANGES_REQUESTED** [review]

**Review by raisedadead — APPROVED** [review]

**Review by marsonya — APPROVED** [review]

**Comment by nhcarrigan** [comment]
@raisedadead before landing this, I asked Quincy to give it a once-over as well to make sure everything is still accurate and up-to-date.

**Comment by raisedadead** [comment]
I will let you merge this instead. Thanks for letting me know.

**Review by QuincyLarson — APPROVED** [review]
This all looks solid. No further edits recommended from me.

**Pull request merged**